### ISSUE TITLE
404 for course in /entity/unrevised with first course page has no current revision yet

### ISSUE THREAD

**Issue by kulla** [serlo/frontend#1337]
In `/entity/unrevised` there is a link to the course https://de.serlo.org/informatik/227404/versionskontrolle-mit-git which has only one course page without any revised revision yet

```graphql
{
  uuid(id: 227404) {
    __typename
    ... on Course {
      pages {
        id
        currentRevision {
          id
        }
        revisions {
          nodes {
            id
          }
        }
      }
    }
  }
}
```

Result:

```json
{
  "data": {
    "uuid": {
      "__typename": "Course",
      "pages": [
        {
          "id": 227406,
          "currentRevision": null,
          "revisions": {
            "nodes": [
              {
                "id": 227408
              }
            ]
          }
        }
      ]
    }
  }
}
```

Click on the link in `/entity/unrevised` results in a 404 error page. @elbotho Do you have any clue what the desired behavior shall be? Shall I filter out `Course` entities in `unrevisedEntities` and only return `CoursePage`?! Shall it be handled in the frontend?

**Comment by elbotho** [comment]
hihi 1337

**Comment by kulla** [comment]
> hihi 1337

?

**Comment by Entkenntnis** [comment]
elite

**Comment by elbotho** [comment]
> Shall I filter out Course entities in unrevisedEntities and only return CoursePage

~Do you know if courses have to be reviewed atm.?~

~If no, filtering is a good idea imo.~
The problem here is that the frontend does not have a course view. We always load to the first course page instead which fails here.

(PR with fix here #1363 )
